README.md

```diff
@@ -108,6 +108,7 @@ For command management Jennifer now uses [Sam](https://github.com/imdrasil/sam.c
 ```crystal
 require "./your_configuration_folder/*" # with requiring jennifer and her adapter
 require "./migrations/*"
+require "sam"
 load_dependencies "./", "jennifer"
```

Comment:
@forsaken1 Also please change next line to `load_dependencies "jennifer"` - Sam will take care of regular case by its own